mkdocs.1

```diff
@@ -0,0 +1,27 @@
+.TH "MKDOCS" "1" "0.1" "William Moreno Reyes" ""
+.SH "NAME"
+mkdocs \- Create HTML documentation from markdown sources
+.SH "SYNOPSIS"
+mkdocs [help|new|build|serve|gh\-deploy] {options}
+.SH "DESCRIPTION"
+MkDocs is a fast and simple way to create a website from source files written in Markdown, and configured with a YAML configuration file, the documentation  can be hosted anywhere, even in free hosting services like Read the Docs and GitHub Pages.
+.SH "OPTIONS"
+.TP
+\fBhelp\fP
+display this help and exit
+.TP
+\fBnew\fP 
+\fB<name>\fP create a new project with under the \fI<name>\fP directory
+.TP
+\fBbuild\fP
+make a HTML website from sources
+.TP
+\fBserve\fP
+start a server to view in real time the documentation
+.TP
+\fBgh\-deploy\fP
+commit changes to a GitHub Page
+.SH "AUTHOR"
+William Moreno Reyes
```

Comment:
This is the author of the manpage, you can edit everthing you want and add you, or rewritte all the manpage and only let your name here.